include/pmacc/particles/tasks/TaskSendParticlesExchange.hpp

```diff
@@ -118,7 +118,7 @@ namespace pmacc
                 std::cerr << "Send/receive buffer for species " <<
                     ParBase::FrameType::getName() <<
                     " is too small (max: " << maxSize <<
-                    ", direction: " << exchange <<
+                    ", direction: " << exchange << " '" << ExchangeTypeNames{}(exchange) << "'" <<
```

Comment:
I think we should add the proper include for this type as well, since it's not defined in this file.

Reply:
good point, updated

Reply:
might need a git rebase, but the class seems not to be known yet with older GCC (4.9.4):
```
no match for call to ‘(pmacc::ExchangeTypeNames) (uint32_t&)’
```

Reply:
no it is my fault I need to use the `operator[]`